src/components/Input/Input.jsx

```diff
@@ -65,6 +75,8 @@ Input.propTypes = {
   type: PropTypes.string,
   /** The value of the input */
   value: PropTypes.string,
+  /** The prop to get the DOM element of the Button */
```

Comment:
@federicoweber not sure if I still need this here.. :philosoraptor:

Reply:
Yep we can remove this one, `ref` is not a prop 😉.